erd.md

```
@@ -0,0 +1,174 @@
+    reviewed_by  BIGINT,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_report_reporter    FOREIGN KEY (reporter_id) REFERENCES member (id),
+    CONSTRAINT fk_report_reviewed_by FOREIGN KEY (reviewed_by) REFERENCES member (id)
+);
+```
+
+---
+
+## 관계 요약
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)